packages/data/addon/services/metric-name.js

```diff
@@ -29,7 +34,7 @@ export default Service.extend({
    */
   getDisplayName(metricObject) {
     let metricId = get(metricObject, 'metric'),
-        longName = this.getLongName(metricId);
+      longName = this.getLongName(metricId);
```

Comment:
Won't this spacing conflict with our eslint rule?

Reply:
yeah, we need to update eslint config

Reply:
I liked how it was before. Can we not configure prettier to behave the same way eslint does currently?

Reply:
Unfortunately, prettier is not very configurable.

Reply:
Lets discuss this during the navi meeting tomorrow.